java/README.md

```diff
@@ -84,26 +85,23 @@ Other useful gradle developer commands:
 * `./gradlew :client:test` to run client unit tests
 * `./gradlew spotlessCheck` to check for codestyle issues
 * `./gradlew spotlessApply` to apply codestyle recommendations
+* `./gradlew :examples:run` to run client examples
 * `./gradlew :benchmarks:run` to run performance benchmarks
 
 ## Basic Examples
 
 ### Standalone Redis:
 
 ```java
-import glide.Client;
-import glide.Client.SingleResponse;
-
-Client client = new Client();
+import glide.api.RedisClient;
 
-SingleResponse connect = client.asyncConnectToRedis("localhost", 6379);
-connect.await().isSuccess();
+RedisClient client = RedisClient.CreateClient();
 
-SingleResponse set = client.asyncSet("key", "foobar");
-set.await().isSuccess();
+CompletableFuture<Void> setResponse = client.set("key", "foobar");
```

Comment:
I'm guessing this will become accurate when the GET/SET/INFO/PING PR is merged?

Reply:
That is correct. 
No point changing the README a dozen times.